lib/models/asyncapi.js

```diff
@@ -208,18 +209,19 @@ class AsyncAPIDocument extends Base {
   /**
    * Callback used when crawling a schema.
    * @callback TraverseSchemas
-   * @param {Schema} schema which is being crawled 
+   * @alias module:@asyncapi/parser#TraverseSchemas
```

Comment:
This took a lot of time to figure out... 

Problem is the type definition HAVE to be declared inside the global module `"@asyncapi/parser"` for it to know where to find the Schema declaration. I tried to change the type of the Schema variable but was unable to do something along the lines of `* @param {module:@asyncapi/parser.Schema} schema which is being crawled`  but with no luck. 

So I ended up with the following definition:

```suggestion
   * @callback module:@asyncapi/parser.TraverseSchemas
   * @param {Schema} schema which is being crawled
```